conf.py

```diff
@@ -253,69 +253,57 @@
-                "hadoop_long_version": "3.3.1",
-                "aws_java_sdk_bundle": "1.11.901",
-                "azure_storage": "7.0.1",
-                "azure_keyvault_core": "1.0.0",
-                "jackson_dataformat_xml": "2.12.3",
-                "stax2_api": "4.2.1",
-                "woodstox_core": "6.2.1",
-                "vector": "0.31.0",
-            },
-            {
-                "product": "3.3.0",
-                "spark": "3.3.0",
+                "product": "3.4.0",
+                "spark": "3.4.0",
                 "java-base": "11",
-                "python": "39",
+                "python": "3.11",
                 "hadoop_short_version": "3",
-                "hadoop_long_version": "3.3.3",
-                "aws_java_sdk_bundle": "1.11.1026",
+                "hadoop_long_version": "3.3.4",
+                "aws_java_sdk_bundle": "1.12.262",
                 "azure_storage": "7.0.1",
                 "azure_keyvault_core": "1.0.0",
-                "jackson_dataformat_xml": "2.13.3",
+                "jackson_dataformat_xml": "2.14.2",
                 "stax2_api": "4.2.1",
-                "woodstox_core": "6.2.1",
+                "woodstox_core": "6.5.0",
                 "vector": "0.31.0",
+                "jmx_exporter": "0.20.0",
             },
+            # required for a customer
             {
-                "product": "3.3.0-java17",
-                "spark": "3.3.0",
+                "product": "3.4.0-java17",
+                "spark": "3.4.0",
                 "java-base": "17",
-                "python": "39",
+                "python": "3.11",
                 "hadoop_short_version": "3",
-                "hadoop_long_version": "3.3.3",
-                "aws_java_sdk_bundle": "1.11.1026",
-                "azure_storage": "7.0.1",
-                "azure_keyvault_core": "1.0.0",
-                "jackson_dataformat_xml": "2.13.3",
-                "stax2_api": "4.2.1",
-                "woodstox_core": "6.2.1",
+                "hadoop_long_version": "3.3.4",  # https://github.com/apache/spark/blob/1db2f5c36b120c213432fc658c9fd24fc73cb45e/pom.xml#L122
+                "aws_java_sdk_bundle": "1.12.262",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/3.3.4
+                "azure_storage": "7.0.1",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-azure/3.3.4
+                "azure_keyvault_core": "1.0.0",  # https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage/7.0.1
+                "jackson_dataformat_xml": "2.14.2",  # https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.13/3.4.0
+                "stax2_api": "4.2.1",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.14.2
+                "woodstox_core": "6.5.0",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.14.2
                 "vector": "0.31.0",
```

Comment:
Missing. Without it, the bake build fails.
> "jmx_exporter": "0.20.0",

Reply:
https://github.com/stackabletech/docker-images/pull/475/commits/08a03beafbd52381eb416863d9bb04990d9bc6db